internal/app/command.go

```diff
@@ -25,6 +27,27 @@ func (c *command) String() string {
 	return c.Cmd + " " + strings.Join(c.Args, " ")
 }
 
+// runs exec command with retry
+func (c *command) retryExec(attempts int) exitStatus {
+	var result exitStatus
+	var errMsg string
+	err := retry(attempts, 2*time.Second, func() (err error) {
```

Comment:
I don't think we need this many levels of wrapping. We could just call the exec function here in a loop and parse the exit code directly. If the goal is to have something more generic we can simply use something like https://github.com/avast/retry-go

Reply:
ok, I went easier with the loop inside of retryExec
and also added math.Pow to time sleep there